Modules/Sources/WordPressUI/Views/ListFooterView.swift

```diff
@@ -0,0 +1,50 @@
+import SwiftUI
+
+public struct ListFooterView: View {
+    public enum State {
+        case loading
+        case failure
+    }
+
+    let state: State
+    var onRetry: (() -> Void)?
+
+    public init(_ state: State) {
+        self.state = state
+    }
+
+    public func onRetry(_ closure: (() -> Void)?) -> ListFooterView {
+        var copy = self
+        copy.onRetry = closure
+        return copy
+    }
+
+    public var body: some View {
+        Group {
+            contentView
+        }
```

Comment:
Is the purpose of using `Group` here that we need to make the UI look a certain way?

Reply:
Oh I think I just forget to remove it when I moved other code. I'll remove it.

Reply:
Removed the `Group`:

<img width="320" alt="Screenshot 2025-05-07 at 9 11 57 AM" src="https://github.com/user-attachments/assets/9ba6909b-f7bc-418b-865d-218fcb97ee92" />